internal/cmp/data_source_active_monitor.go

```diff
@@ -0,0 +1,48 @@
+// (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+
+package cmp
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/HewlettPackard/hpegl-vmaas-cmp-go-sdk/pkg/client"
```

Comment:
Please note that for every resource, there should be one data source definition. We cant have multiple of them. What is difference this one and the other one ?

Reply:
updated